hercules/amr_wind_standin.py

```diff
@@ -184,12 +184,16 @@ def run(self):
             logger.info("Calculating simulation time: %.1f" % sim_time_s)
 
             # Compute the turbine power using a simple formula
+            if self.message_from_server is not None:
+                yaw_angles = self.message_from_server[-self.num_turbines :]
+            else:
+                yaw_angles = None
             (
                 amr_wind_speed,
                 amr_wind_direction,
                 turbine_powers,
```

Comment:
We need to decide what units power is in.  It seems to be mostly in kW, I think, but this seems to give power back in watts from Floris?

Reply:
After discussion, moving forward with power being specified in kW across all modules. I'll update the FLORIS stand-in to reflect this.

Reply:
@genevievestarke I've now updated the Floris stand-in code to convert directly to kW on receiving the powers from Floris (tests also updated to match)